OCR/frontend/api/api.ts

```diff
@@ -1,18 +1,20 @@
-import { Page } from "../src/contexts/FilesContext";
-import { ImageToTextResponse } from "./types/types";
+import { ImageToTextArgs, ImageToTextResponse } from "./types/types";
 
-export const AddFormData = async (args: Page): Promise<ImageToTextResponse | null> => {
+export const ImageToText = async (args: ImageToTextArgs): Promise<ImageToTextResponse | null> => {
 
     const { sourceImage, templateImage, fieldNames } = args;
-    const form = new FormData();
-    form.append("source_image", sourceImage);
-    form.append("segmentation_template", templateImage);
-    form.append("labels", JSON.stringify(fieldNames));
-    form.append("", "");
-    console.log(args)
+    const form = new URLSearchParams({
+        source_image: sourceImage,
+        segmentation_template: templateImage,
+        labels: JSON.stringify(fieldNames),
+      });
+      
     try {
         const response = await fetch("http://localhost:8000/image_to_text/", {
```

Comment:
extract `http://localhost:8000` into a environment variable, name it `API_URL`
https://vitejs.dev/guide/env-and-mode

Reply:
updated